package co.edu.uniquindio.taller2.tests;

import org.junit.Assert;

import org.junit.Test;

import co.edu.uniquindio.taller2.model.Banco;
import co.edu.uniquindio.taller2.model.CuentaException;

public class TestBankTrue {

	@Test
	public void test() {
		Banco bank = new Banco("Bancolombia", "Cra 9 # 23 - 01", "AB-001");
		try {
			bank.agregarCuentaAhorros("A001", 0, 0);
		} catch (CuentaException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
	}

}
